inc/meta-box-html.php

```diff
@@ -459,8 +459,9 @@ protected function date( $taxonomy, $tax_nice_name, $multiples = false, $require
 					if ( is_numeric( $term->name ) ) {
 						$natdate = date( 'j F, Y', intval( $term->name ) );
 	?>
-			  <span><a id='<?php echo sanitize_text_field( $taxonomy ) ?>-check-num-<?php echo sanitize_text_field( $i ) ?>' class='ntdelbutton <?php echo sanitize_text_field( $term->name ) ?>'><?php echo sanitize_text_field( $term->name ) ?></a>&nbsp;<?php echo $natdate ?></span>
+			  <span><a id='<?php echo sanitize_text_field( $taxonomy ) ?>-check-num-<?php echo sanitize_text_field( $i ) ?>' class='datedelbutton <?php echo esc_attr( $term->name ) ?>'><?php echo sanitize_text_field( $term->name ) ?></a><?php echo $natdate ?></span>
```

Comment:
`esc_attr` is [better suited for html attributes](http://codex.wordpress.org/Function_Reference/esc_attr). `sanitize_text_field` is probably a bit overkill for html attributes, I think I learned that while writing the form and changed some but not all of them.